app/questionnaire/router.py

```diff
@@ -219,15 +219,20 @@ def _get_return_to_location_url(
         if not return_to:
             return None
 
-        if return_to == "calculated-summary":
-            if return_to_block_id in routing_path:
-                return url_for(
-                    "questionnaire.block",
-                    list_item_id=location.list_item_id,
-                    block_id=return_to_block_id,
-                    _anchor=return_to_answer_id,
-                )
-            return None
+        if (
+            return_to == "calculated-summary"
+            and return_to_block_id in routing_path
```

Comment:
This check isn't needed as it will be covered by the `can_access_location` check.

Reply:
Removed now 👍